### ISSUE TITLE
Consider adding features around RTools for Windows integrated terminals

### ISSUE THREAD

**Issue by juliasilge** [posit-dev/positron#5358]
RStudio [does a lot of careful work](https://github.com/rstudio/rstudio/blob/b88826c72304ae881e714061d3370f63dee1982f/src/cpp/session/modules/build/SessionBuildEnvironment.cpp#L100) to provide and use RTools on Windows, but Positron doesn't have any of that currently. This means, for example, that `make` is probably not available by default as outlined in https://github.com/posit-dev/positron/discussions/5326#discussion-7461757. This applies mostly to R package developers working on Windows.

For now, folks are on their own when it comes to RTools but let's consider adding features (like finding and/or prompting to install RTools) for a better Windows experience.

**Comment by DavisVaughan** [comment]
I would encourage people to use rig. It does all the hard work for you
https://github.com/r-lib/rig/blob/532d6e5d330a4b900cc26ff99be5559815b5a211/src/help-windows.in#L258

TLDR:

```bash
rig system rtools add
```

**Comment by petetronic** [comment]
We can start by documenting guidance in the official Positron docs, and in the future look into better detection + fixing the issue.

**Comment by juliasilge** [comment]
The documentation is now updated at: <https://positron.posit.co/start.html#windows-prerequisites>

In the future, we can explore how to detect and fix Rtools setup.

**Comment by serkor1** [comment]
If I may add something here to the issue here, which is an extension to the discussion  https://github.com/posit-dev/positron/discussions/5331 about user-friendliness.

I have played around with `Makefiles` on Linux too. If I run my command

```make
check:
	R CMD build .
	R CMD check $(TARBALL)
	rm -f $(TARBALL)
	rm -rf $(PKGNAME).Rcheck
```

I get the following error:

```
R CMD build .
make: R: Permission denied
make: *** [Makefile:40: check] Error 127
```

So I need to run it as `sudo make check`, to circumvent this. However, when I do that - I get the following error:

```
Error in loadVignetteBuilder(pkgdir, TRUE) : 
  vignette builder 'knitr' not found
Execution halted
make: *** [Makefile:40: check] Error 1
```

I can run this without any sudo privileges and issues using RStudio.

**Comment by juliasilge** [comment]
No worries @serkor1! If you are coming up against something slightly different (i.e. using `make` on Linux), then it is helpful to open a separate discussion rather than add to an existing issue on a different topic (i.e. getting RTools on Windows). If you can open a new discussion and outline what you are experiencing in these two Linux environments, that would be helpful!
